src/index.ts

```diff
@@ -1,5 +1,3 @@
 export * from './types';
-export * from './client/addon/withCreevey';
-export * from './client/addon/readyForCapture';
```

Comment:
That breaks imports in a `.storybook/preview.js` and you will loose ability to import `readyForCapture`. I think we need another approach.

Reply:
But deep imports from `creevey/client/addon/*` are still available. What if we add an index file there and offer to import everything addon-specific from `creevey/client/addon`?

Reply:
Really? I thought because of `exports` field in package.json you can't import anything except root file
https://nodejs.org/api/packages.html#main-entry-point-export
> When the ["exports"](https://nodejs.org/api/packages.html#exports) field is defined, all subpaths of the package are encapsulated and no longer available to importers. For example, require('pkg/subpath.js') throws an [ERR_PACKAGE_PATH_NOT_EXPORTED](https://nodejs.org/api/errors.html#err_package_path_not_exported) error.

Maybe we can define `exports` as an object:
```json
{
  "exports": {
    ".": "./lib/cjs/index.js",
    "./addon/*": "./lib/cjs/client/addon/*.js"
  }
}
```
We can even split cjs and esm exports in that way, but maybe later

Reply:
My "deep imports" suggestion really don't work because of built files structure.

Reply:
Moved to the separate PR https://github.com/creevey/creevey/pull/231.